packages/password/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "version": "1.0.0",
+  "description": "Password generation utilities",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run test",
+    "generate:html": "node html-testing/generate.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/duckduckgo/duckduckgo-autofill.git"
+  },
+  "keywords": [
+    "duckduckgo",
+    "password"
+  ],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/duckduckgo/duckduckgo-autofill/issues"
+  },
+  "homepage": "https://github.com/duckduckgo/duckduckgo-autofill#readme"
```

Comment:
Is it worth using `/password/docs/password.md` or otherwise link to that doc from the main README?

Reply:
Yep, it'll 404 for now (because I'll point to `main`), but that won't be an issue once it's released